MainWindow.xaml.cs

```diff
@@ -26,7 +27,7 @@ public partial class MainWindow
         /// </summary>
         private readonly StalkbotClient _client;
         private readonly NotifyIcon _notifyIcon;
-        private const string Version = "1.2";
+        private static readonly Version _version = Version.Parse("1.2");
```

Comment:
Naming Conventions :trogi:
```suggestion
        private static readonly Version Version = Version.Parse("1.2");
```